docs/release.md

```diff
@@ -9,17 +9,56 @@
 - The API ML is a part of the Zowe PAX file that is packaged by builds of the [zowe-install-packaging](https://github.com/zowe/zowe-install-packaging/) repository.
 - When a new release is available for a release candidate, the [zowe-install-packaging/manifest.json.template file on the RC branch](https://github.com/zowe/zowe-install-packaging/blob/rc/manifest.json.template) needs to be updated so the `org.zowe.apiml.sdk.*` components have the proper version.
   - [Zowe Release Process](https://github.com/zowe/community/blob/master/Technical-Steering-Committee/release.md)
-- If you need to publish a PR version of the API ML, use the [specific release action](https://github.com/zowe/api-layer/actions/workflows/specific-release.yml) and use the current snapshot version as new version and the currently released version with an added PR as the current version {VERSION}-PR-{PR_NUMBER} and run from the branch for the PR you want to release. Example:
-  - Current Version: 1.22.1-PR-1475
-  - New Version: 1.22.2-SNAPSHOT
-  - Branch: apiml/GH1456/specific_release
+- If you need to publish a PR version of the API ML, use the [PR snapshot workflow](https://github.com/zowe/api-layer/actions/workflows/pull-request-snapshot-release.yml) and use the current snapshot version as new version and the currently released version with an added PR as the current version {VERSION}-PR-{PR_NUMBER} and run from the branch for the PR you want to release. Example:
+  - Pull request: PR-123
+  - Branch: apiml/GH123/my_branch
+- If you need to publish images of the API ML for testing purposes, use the [create image workflow](https://github.com/zowe/api-layer/actions/workflows/create-image-without-release.yml)
+  - Service: all
+  - Branch: my-branch-for-testing
+
+## Releasing API ML for a Zowe Release
+
+When there is a new minor release of Zowe, there should be a new patch release of the API ML from the `master` branch and a new minor snapshot version created.
+This is done in two steps:
+ 
+1. Release the binaries with the [binary specific release workflow](https://github.com/zowe/api-layer/actions/workflows/binary-specific-release.yml).
+    `release_version` is the version that will be released. This should be a new patch version. For example, if `master` is currently on version 1.27.1, `release_version` would be 1.27.2
+    `new_version` should be a `SNAPSHOT` version with a new minor version. For example, if `master` is currently on version 1.27.1, `new_version` would be 1.28.0-SNAPSHOT
```

Comment:
What if there is already 1.28.0-SNAPSHOT because it was published with 1.27.0 release. Would it be better to look for the latest SNAPSHOT ?

Reply:
For the binaries it wouldn't present an issue, there just would be a new SNAPSHOT published version